onnxruntime/core/providers/coreml/builders/impl/resize_op_builder.cc

```diff
@@ -215,12 +221,12 @@ bool ResizeOpBuilder::IsOpSupportedImpl(const Node& node, const OpBuilderInputPa
       float scale_w = scales[3];
 
       // Onnx spec requires scale to be a positive float, so we are not checking that here
-      if (roundf(scale_h) != scale_h) {
+      if (round(scale_h) != scale_h) {
```

Comment:
Why? round is for doubles,

Reply:
Sorry, I am just trying to see why `roundf()`it still shows a build error not declared in the scope. changed it back.